src/js/errorHandling.js

```diff
@@ -1,16 +1,28 @@
 // sprawdzanie połączenia z internetem
 const main = document.querySelector(".main");
-window.addEventListener("online", function() {
-    const node = main.firstElementChild
-    main.removeChild(node);
-})
-window.addEventListener("offline", function() {
-    const node = document.createElement("div");
-    const textnode = document.createTextNode("brak dostępu do sieci");
-    const child = document.querySelector(".main__box1"); 
-    node.classList.add("no-connection") 
-    node.appendChild(textnode);                        
-    main.insertBefore(node, child); 
-})
-
+const ifoffline = () => {
+    if(!navigator.onLine) {
+        const node = document.createElement("div");
+        const textnode = document.createTextNode("brak dostępu do sieci");
+        const child = document.querySelector(".main__box1"); 
+        node.classList.add("no-connection") 
+        node.appendChild(textnode);                        
+        main.insertBefore(node, child); 
+    }
+}     
+ifoffline(); //wywołuję tą funkcję tylko raz na początku aby sprawdzić czy jest offline aby dodać powiadomienie. W mojej przeglądarce listener nie obsługiwał tego na początku.
```

Comment:
czemu to jest chujwiecotoCase a nie camelCase?

Reply:
Hahahahah xD